src/awkward/_connect/julia/utils.py

```diff
@@ -158,12 +156,12 @@ def init_julia(julia_project=None, quiet=False, julia_kwargs=None, return_aux=Fa
             )
 
     if julia_initialized and julia_activated_env != processed_julia_project:
-        Main.eval("using Pkg")
+        julia_main.eval("using Pkg")
```

Comment:
I don't know if it's better to use the same name in Python as we would in Julia. Would it make the code more consistent and readable if for people familiar with Julia conventions?

Reply:
I think that's a good point. Because this code is internal, I'm less worried about that than I am about following our internal style guidelines. Are you happy with that reasoning?

Reply:
yes, thanks!